webtau-http-groovy/src/test/groovy/com/twosigma/webtau/http/HttpGroovyTest.groovy

```diff
@@ -468,6 +471,72 @@ class HttpGroovyTest implements HttpConfiguration {
         assertFalse(Files.exists(requestFile))
     }
 
+    @Test
+    void "appropriate capture files are generated"() {
```

Comment:
should we be explicit that it is about `doc` related capture?

Reply:
Done.